src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import {Routes,Route} from "react-router";
```

Comment:
Use the correct usage of spaces
```suggestion
import { Routes, Route } from "react-router";
```